src/main/java/frc/robot/subsystems/drivetrain/commands/AssistedDrive.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.Robot;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import java.util.function.DoubleSupplier;
+
+public class AssistedDrive extends HolonomicDrive {
```

Comment:
You can use the older version of the SwerveDrive, it does the same, just internally ([reference](https://github.com/Galaxia5987/Swerve-2022/blob/747a8b57d92b5ead8d64e0e36580ef1575835fce/src/main/java/frc/robot/subsystems/drivetrain/SwerveDrive.java#L88))